tools/topology/topology2/production/tplg-targets-ace3.cmake

```diff
@@ -2,6 +2,16 @@
 
 # Array of "input-file-name;output-file-name;comma separated pre-processor variables"
 list(APPEND TPLGS
+# MOCKUP topology for PTL
```

Comment:
why are these mockup? Looks like actual codec topologies to me

Reply:
Ahh yes. I forget this every single time.

Reply:
@bardliao I do wonder should we keep having the mocks for every gens going forward in upstream? Do these still serve a purpose in upstream? 

Given these are upstream Linux, ok to add the topologies if only for consistency (topologies referenced by upstream Linux kernel should be available in SOF tplg upstream).